admin_board_view/views.py

```diff
@@ -391,6 +391,12 @@ def export_sale_transactions(request):
                 name = "pin betaling" if export_type == "pin" else t.transaction_id
                 response_string += f'"",8002,Mongoose - {name},9,{"{:.2f}".format(t.transaction_sum)},""\n'
 
+                # Add transaction fee row for each mollie payment
+                if export_type == "mollie":
+                    response_string += (
+                        '"",5007,Mongoose transaction fee 0.39 x 1,21,0.39,TRX\n'
```

Comment:
See above

Reply:
The TRX fees should be summed up too, so instead of two rows for 5007, it should be one saying x2 and value 0,78